### PR TITLE
Merge release 1.46.1 to develop

### PR DESCRIPTION
**Pull request by fluiddot**
Brings [release 1.46.1 changes](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3145) back to `develop` branch.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

**Comment by fluiddot** [comment]
I wasn't sure about what commit should be referenced in the Gutenberg submodule, I decided to use the hash of the [merge commit](https://github.com/WordPress/gutenberg/commit/b70ef682243cd888060cd7aa7e13686708da0479) ("Mobile Release v1.46.1"), is this correct?

**Comment by ceyhun** [comment]
> I wasn't sure about what commit should be referenced in the Gutenberg submodule, I decided to use the hash of the merge commit ("Mobile Release v1.46.1"), is this correct?

I think it's fine as long as gutenberg-mobile is pointing to a commit in gutenberg that is on master branch.

**Comment by fluiddot** [comment]
> > I wasn't sure about what commit should be referenced in the Gutenberg submodule, I decided to use the hash of the merge commit ("Mobile Release v1.46.1"), is this correct?
> 
> I think it's fine as long as gutenberg-mobile is pointing to a commit in gutenberg that is on master branch.

Now the Gutenberg reference points to the same value we currently have in `develop` because it's newer than the previous one I pushed in this PR.

File: package-lock.json [hunk 1]
```diff
@@ -2915,6 +2915,211 @@
         "esutils": "^2.0.2"
       }
     },
+    "@babel/preset-typescript": {
```
Comment (fluiddot) [inline package-lock.json:2918]: I was surprised about this PR adds this but I checked in `develop` and looks like the `package-lock.json` is out-dated. I ran `npm install` in the latest commit on `develop` and the file was modified 🤷‍♂️ .

**Comment by fluiddot** [comment]
This PR is no longer need as the changes from `1.46.1` (as well as `1.46.2`) beta fixes were included when merging [`1.47.0` version](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3181).

**Pull request closed without merging**